infra/main.bicep

```diff
@@ -662,7 +664,6 @@ module adminweb './app/adminweb.bicep' = if (hostingModel == 'code') {
       BACKEND_URL: 'https://${functionName}.azurewebsites.net'
       DOCUMENT_PROCESSING_QUEUE_NAME: queueName
       FUNCTION_KEY: clientKey
-      ORCHESTRATION_STRATEGY: orchestrationStrategy
```

Comment:
I believe the Teams chat goes through the Function and it would need the Orchestration Strategy

Reply:
ahh.. good spot. that was accidental.